java8/build/Dockerfile

```diff
@@ -6,8 +6,14 @@ WORKDIR /
 
 RUN rm -rf /var/runtime /var/lang && \
   curl https://lambci.s3.amazonaws.com/fs/java8.tgz | tar -zx -C / && \
-  yum install -y --releasever=latest java-1.8.0-openjdk-devel-1.8.0.181
+  yum install -y --releasever=latest java-1.8.0-openjdk-devel-1.8.0.181 && \
+  mkdir /opt/gradle && curl -L -o gradle.zip https://services.gradle.org/distributions/gradle-5.2-bin.zip && \
+  unzip -d /opt/gradle gradle.zip && rm gradle.zip && mkdir /opt/maven && \
+  curl -L http://mirror.metrocast.net/apache/maven/maven-3/3.6.0/binaries/apache-maven-3.6.0-bin.tar.gz -o maven.tar.gz && \
+  tar -C /opt/maven -zxf maven.tar.gz && rm maven.tar.gz
+
+ENV PATH="${PATH}:/opt/gradle/gradle-5.2/bin:/opt/maven/apache-maven-3.6.0/bin/"
```

Comment:
Any reason not to put them at the start of the path? I feel like that would lead to less issues if Lambda installed a maven or gradle, for example.

And minor nit: `.../bin`, not `.../bin/` for the maven piece.

Reply:
There wasn't really a reason. I can flip the order.